packages/front-end/components/DataViz/DataVizConfigPanel.tsx

```diff
@@ -120,368 +122,501 @@ export default function DataVizConfigPanel({
         <SelectItem value="line">Line</SelectItem>
         <SelectItem value="area">Area</SelectItem>
         <SelectItem value="scatter">Scatter</SelectItem>
+        <SelectItem value="big-value">Big Value</SelectItem>
       </Select>
 
       <Separator size="4" my={"2"} />
 
-      <Select
-        label="X Axis"
-        value={dataVizConfig.xAxis?.fieldName ?? ""}
-        setValue={(v) => {
-          if (!v) return;
-          const type = getInferredFieldType(v);
-          onDataVizConfigChange({
-            ...dataVizConfig,
-            xAxis: {
-              fieldName: v,
-              type,
-              sort:
-                type !== "string" ? "asc" : dataVizConfig.xAxis?.sort || "none",
-              // TODO: infer date aggregation unit based on data
-              dateAggregationUnit: "day",
-            },
-          });
-        }}
-        size="2"
-        placeholder="Select X Axis"
-      >
-        {axisKeys.map((key) => (
-          <SelectItem key={key} value={key}>
-            {key}
-          </SelectItem>
-        ))}
-      </Select>
-
-      {dataVizConfig.xAxis && (
-        <Flex direction="column" gap="2">
-          <Flex direction="row" justify="between" align="center">
-            <Text as="label" size="2" mr="2" style={{ flex: 1 }}>
-              Type
-            </Text>
-            <Select
-              style={{ flex: 1 }}
-              value={dataVizConfig.xAxis.type}
-              setValue={(v) => {
-                if (!v || !dataVizConfig.xAxis) return;
-                onDataVizConfigChange({
-                  ...dataVizConfig,
-                  xAxis: {
-                    ...dataVizConfig.xAxis,
-                    type: v as "string" | "number" | "date",
+      {dataVizConfig.chartType === "big-value" ? (
+        <>
+          <Select
```

Comment:
Can we update the new Selects to also have `size="2"`? I think for secondary configuration (basically everything except Name & Graph Type) use the smaller size